scripts/devops_tasks/Smoke-Tests-Release.ps1

```diff
@@ -0,0 +1,42 @@
+#Requires -Version 7.0
+
+Param(
+    [string] $serviceDirectory
+)
+
+$repoRoot = Resolve-Path "$PSScriptRoot/../../"
+
+Push-Location $repoRoot
+
+# create a smoketests directory
+$smoketestsDir = Join-Path $repoRoot eng scripts smoketest 
+Write-Host "Creating a new directory for smoketests at $smoketestsDir"
+New-Item -Path $smoketestsDir -ItemType Directory
+
+# Run smoketests python script to create requirements.txt
+Write-Host "Creating requirements.txt in the smoketest folder"
+virtualenv env
```

Comment:
Is this going to be installed by default on the CI agents?

Reply:
yes - it's installed by default